chains/substrate/connection_test.go

```diff
@@ -28,7 +28,7 @@ func TestConnect_QueryStorage(t *testing.T) {
 
 func TestConnect_SubmitTx(t *testing.T) {
 	// Create connection with Alice key
-	conn := NewConnection(TestEndpoint, &TestKeyringPairAlice)
+	conn := NewConnection(TestEndpoint, "Alice", AliceKey)
```

Comment:
nit: "alice" is misleading since the second parameter here is `chain.name` which references a blockchains name.

Reply:
The alternative is "substrate" and "substrate2", this seems more informative